querydb/src/main/scala/io/joern/scanners/php/PhpJoern.scala

```diff
@@ -0,0 +1,144 @@
+package io.joern.scanners.php
+
+import io.joern.console.*
+import io.joern.dataflowengineoss.language.*
+import io.joern.dataflowengineoss.queryengine.EngineContext
+import io.joern.macros.QueryMacros.*
+import io.joern.scanners.*
+import io.shiftleft.codepropertygraph.generated.Operators
+import io.shiftleft.semanticcpg.language.*
+
+object PhpJoern extends QueryBundle {
+
+  private val PHPJOERN_TAG: String = "phpjoern"
+  private val PHPJOERN_EXT_AUT: String = "yichao"
+
+  implicit val resolver: ICallResolver = NoResolve
+
+  @q
+  def sqli()(implicit context: EngineContext): Query =
+    Query.make(
+      name = "phpjoern_sqli",
+      author = PHPJOERN_EXT_AUT,
+      title = "CWE-89(SQL Injection): A parameter is used in an insecure sqli related func call.",
+      description = """
+          |An attacker controlled parameter is used in an insecure sqli related func call.
+          |
+          |If the parameter is not validated and sanitized, this is a sql injection.
+          |""".stripMargin,
+      score = 5,
+      withStrRep({ cpg =>
+        // $_REQUEST["foo"], $_GET["foo"], $_POST["foo"]
+        // are identifier (at the moment)
+        def source =
+          cpg.call.name(Operators.assignment).argument.code(".*_(REQUEST|GET|POST).*")
```

Comment:
This might be simpler?
```suggestion
          cpg.assignment.source.code(".*_(REQUEST|GET|POST).*")
```

Reply:
Same suggestion for below